### PR TITLE
[15.0][MIG] point_of_sale

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
.

### CODE REVIEW

**Comment by dansanti** [comment]
hello!, works fine

**Review by fuentes73 — APPROVED** [review]

File: openupgrade_scripts/scripts/point_of_sale/15.0.1.0.1/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+point_of_sale / account.payment          / force_outstanding_account_id (many2one): NEW relation: account.account
+point_of_sale / account.payment          / pos_payment_method_id (many2one): NEW relation: pos.payment.method
+# NOTHING TO DO: new relations
+
+point_of_sale / account.payment          / pos_session_id (many2one)     : NEW relation: pos.session
+point_of_sale / pos.session              / bank_payment_ids (one2many)   : NEW relation: account.payment
+# NOTHING TO DO: new relation
+
+point_of_sale / barcode.rule             / type (False)                  : selection_keys is now '['alias', 'cashier', 'client', 'discount', 'expiration_date', 'location', 'location_dest', 'lot', 'package', 'package_type', 'packaging_date', 'price', 'product', 'quantity', 'use_date', 'weight']' ('['alias', 'cashier', 'client', 'discount', 'location', 'lot', 'package', 'price', 'product', 'weight']')
+# NOTHING TO DO: same selection options in this module
```
Comment (remytms) [inline openupgrade_scripts/scripts/point_of_sale/15.0.1.0.1/upgrade_analysis_work.txt:26]: ~~It seams that `expiration_date`, `location_dest`, `package_type`, `packaging_date`, `quantity` and `use_date` are no longer valid value for this field. Do they have been moved to other modules ? Is there a replacement value for them ?~~

Comment (remytms) [reply openupgrade_scripts/scripts/point_of_sale/15.0.1.0.1/upgrade_analysis_work.txt:26]: Whoups, sorry. I read it the wrong way. :p

**Pull request merged**